generic3g/MAPL_Generic.F90

```diff
@@ -40,15 +40,17 @@ module mapl3g_Generic
    use :: esmf, only: ESMF_ConfigGet
    use :: esmf, only: ESMF_HConfig
    use :: esmf, only: ESMF_HConfigIsDefined
-   use :: esmf, only: ESMF_HConfigAsString
    use :: esmf, only: ESMF_SUCCESS
    use :: esmf, only: ESMF_Method_Flag
    use :: esmf, only: ESMF_STAGGERLOC_INVALID
    use :: esmf, only: ESMF_StateIntent_Flag
    use :: esmf, only: ESMF_STATEINTENT_IMPORT, ESMF_STATEINTENT_EXPORT, ESMF_STATEINTENT_INTERNAL
    use :: esmf, only: ESMF_TypeKind_Flag, ESMF_TYPEKIND_R4
+   use :: esmf, only: ESMF_KIND_I4, ESMF_KIND_I8, ESMF_KIND_R4, ESMF_KIND_R8
    use :: esmf, only: ESMF_StateItem_Flag, ESMF_STATEITEM_FIELD, ESMF_STATEITEM_FIELDBUNDLE
    use :: esmf, only: ESMF_STATEITEM_STATE, ESMF_STATEITEM_UNKNOWN
+!   use hconfig3g
+   use mapl3g_hconfig_get
```

Comment:
Why isn't this `mapl3g_hconfig_utils`?

Reply:
I saw some places where the **package** module had the mapl3g prefix and some where it did not have the prefix. So this could be `use mapl3g_hconfig_utils` or `use hconfig_utils`. Will fix in morning.